environment.yml

```diff
@@ -20,7 +20,6 @@ dependencies:
   - python>=3.9.2
   - pytorch=1.10.0=py3.9_cuda10.2_cudnn7.6.5_0
   - pyyaml>=5.4.1
-  - ruamel_yaml>=0.15.80
```

Comment:
You remove it but call it in other files

Reply:
It is called in deprecated files like gdl_hyperopt_template.py and data_analysis.py. These files are either waiting to be reimplemented with hydra (#243) (ie without ruamel_yaml) or we haven't decided if we keep them or not (#138).